src/ol/layer/VectorImage.js

```diff
@@ -2,10 +2,13 @@
  * @module ol/layer/VectorImage
  */
 import BaseVectorLayer from './BaseVector.js';
+import {assign} from '../obj.js';
 import CanvasVectorImageLayerRenderer from '../renderer/canvas/VectorImageLayer.js';
 
 /**
  * @typedef {import("./BaseVector.js").Options} Options
+ * @property {number} [imageRatio=1] Ratio by which the rendered extent should be larger than the
+ * viewport extent. A larger ratio avoids cut images during panning, but will cause a decrease in performance.
```

Comment:
It looks like adding a property to a type with this syntax is not supported by TypeScript.  See #9263.